abc.py

```diff
@@ -0,0 +1,15 @@
+from search_engine_parser import BaiduSearch
```

Comment:
What is this file? and why is name abc.py?

Reply:
My bad, it's unnecessary. I was just testing out the Baidu class and forgot to remove it. Will remove in the next commit.